### PR TITLE
Feature/maps

### PR DESCRIPTION
**Pull request by vlach1989**
- Export React redux context
- add actions, reducers and selectors needed in docs
- tests

### CODE REVIEW

File: src/state/Maps/actions.js [hunk 1]
```diff
@@ -24,6 +26,77 @@ import SelectionsAction from '../Selections/actions';
...
+	return (dispatch, getState) => {
+		if (!mapState) {
+			dispatch(commonActions.actionGeneralError(`No map state given`));
+		} else if (!mapState.key) {
+			dispatch(
+				commonActions.actionGeneralError(`Undefined mapKey for map ${mapState}`)
+			);
+		} else {
+			dispatch(actionAddMap(mapState));
+			dispatch(use(mapState.key, null, null));
```
Comment (vdubr) [inline src/state/Maps/actions.js:43]: Add to documentation when "use" is calling.

I mean to functions addMap, addMapToSet, addMapSet

Comment (vdubr) [reply src/state/Maps/actions.js:43]: And remove...

File: src/state/Maps/reducers.js [hunk 2]
```diff
@@ -31,6 +31,72 @@ const setActiveMapKey = (state, mapKey) => {
...
+ * @return {Object} Updated state
+ */
+const addMapToSet = (state, mapKey, mapSetKey) => {
+	if (mapKey && mapSetKey && state.sets[mapSetKey]) {
+		return {
+			...state,
+			sets: {
+				...state.sets,
+				[mapSetKey]: {
+					...state.sets[mapSetKey],
```
Comment (vdubr) [inline src/state/Maps/reducers.js:88]: What if mapSetKey does not exists in state.sets.

File: src/state/Maps/actions.js (hunk 1 above)
Comment (vlach1989) [reply src/state/Maps/actions.js:43]: > Add to documentation when "use" is calling.
> 
> I mean to functions addMap, addMapToSet, addMapSet

Where?

Comment (vlach1989) [reply src/state/Maps/actions.js:43]: > And remove...

What?

File: src/state/Maps/reducers.js (hunk 2 above)
Comment (vlach1989) [reply src/state/Maps/reducers.js:88]: it is solved in line 82

File: src/state/Maps/actions.js (hunk 1 above)
Comment (vlach1989) [reply src/state/Maps/actions.js:43]: https://github.com/gisat-panther/ptr-state/pull/87/commits/2b603df1cdd9bd5c250b7f746fa36d57075979ba

**Review by vdubr — APPROVED** [review]

**Review by tmdmc — APPROVED** [review]

**Pull request merged**